User.Service/User.Service.Test/IntegrationTests/IntegrationTestWebAppFactory.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Xunit;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.AspNetCore.Mvc.Testing;
+using User.Service.DLL.Context;
+using User.Service.API;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace User.Service.Test.IntegrationTests;
+
+public class IntegrationTestWebAppFactory : IAsyncLifetime
+{
+    private const string _connectionString = "test";
```

Comment:
Fyi, you can use appsettings.test.json with test connection string and set environment as `test` for integration tests